### ISSUE TITLE
Help text should be wrapped in p tags

### ISSUE THREAD

**Issue by hbillings** [18F/cms-hitech-apd#1055]
While looking into #1023, I found we've got a bunch of text that is just wrapped in `div`s rather than `p` tags. This isn't semantic (and in some cases, there's probably some ARIA labelling that should happen, also.)

### This task is done when...
 - [x] help text is wrapped in `p` tags
 ~- [ ] help text is semantically linked to its associated content via ARIA attributes~ Striking this for now, as we're going to be doing potentially some destructuring/restructuring of the HTML for stylistic consistency. It'd make more sense to do this as part of that.
